web-test-runner.config.js

```diff
@@ -41,9 +44,33 @@ module.exports = {
     },
     testFramework: {
         config: {
-            timeout: 10000,
+            timeout: 100000,
         },
     },
+    groups: [
+        {
+            name: 'unit',
+            files: 'packages/*/test/*.test.js',
+        },
+        ...vrtGroups,
+        ...packages.reduce((acc, pkg) => {
```

Comment:
This allows us to call `yarn test:focus picker` and only run the tests for the `picker` package, which is quite nice.